<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Car Sharing</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="filterLayout">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>170</y>
      <width>261</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Фильтр</string>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>Марка:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="markBox"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>Модель:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="modelBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QPushButton" name="filterButton">
       <property name="text">
        <string>Найти</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>480</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Система учета проката автомобиле</string>
    </property>
   </widget>
   <widget class="QTreeView" name="CarView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>520</width>
      <height>350</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
    <property name="animated">
     <bool>true</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="headerHidden">
     <bool>false</bool>
    </property>
    <property name="expandsOnDoubleClick">
     <bool>true</bool>
    </property>
    <attribute name="headerCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="headerMinimumSectionSize">
     <number>57</number>
    </attribute>
   </widget>
   <widget class="QLabel" name="loginLabel">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>20</y>
      <width>151</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="logoutButton">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>20</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Выйти</string>
    </property>
   </widget>
   <widget class="QPushButton" name="profileButton">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>50</y>
      <width>261</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Профиль</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="eventBox">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>90</y>
      <width>260</width>
      <height>70</height>
     </rect>
    </property>
    <property name="title">
     <string>Последнее событие</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="eventLabel">
       <property name="text">
        <string>EventName</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="cancelEventButton">
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>810</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>О системе</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="action">
   <property name="text">
    <string>Шаг вперед</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Шаг назад</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
